serveradmin/serverdb/models.py

```diff
@@ -68,6 +69,91 @@ def get_choices(types):
     return zip(*([types] * 2))
 
 
+# TODO: Make validators out of the methods is_ip_address, is_unique and
```

Comment:
I though about this too but I use `ServerInetAttribute` and `Server` model which would introduce a circular dependency. Once could of course pass the QuerySets instead of the models directly but since I am going to refactor this again anyways I decided to make it just work for now this way.

Reply:
Fair enough. 👍🏼